client/my-sites/checkout/checkout-thank-you/test/index.js

```diff
@@ -25,13 +25,26 @@ jest.mock( '@automattic/calypso-products', () => ( {
 jest.mock( 'calypso/lib/analytics/tracks', () => ( {
 	recordTracksEvent: () => null,
 } ) );
-jest.mock( '../domain-registration-details', () => 'component--domain-registration-details' );
-jest.mock( '../google-apps-details', () => 'component--google-apps-details' );
-jest.mock( '../jetpack-plan-details', () => 'component--jetpack-plan-details' );
-jest.mock( '../atomic-store-thank-you-card', () => 'component--AtomicStoreThankYouCard' );
-jest.mock( 'calypso/lib/analytics/page-view-tracker', () => 'PageViewTracker' );
-jest.mock( '../header', () => 'CheckoutThankYouHeader' );
-jest.mock( 'calypso/components/happiness-support', () => 'HappinessSupport' );
+jest.mock( '../domain-registration-details', () => () => 'component--domain-registration-details' );
+jest.mock( '../google-apps-details', () => () => 'component--google-apps-details' );
+jest.mock( '../jetpack-plan-details', () => () => 'component--jetpack-plan-details' );
+jest.mock( '../atomic-store-thank-you-card', () => () => (
+	<div data-testid="AtomicStoreThankYouCard" />
+) );
+jest.mock( 'calypso/lib/analytics/page-view-tracker', () => () => 'PageViewTracker' );
+jest.mock( '../header', () =>
+	jest.fn( ( { children } ) => <div data-testid="checkout-thank-you-header">{ children }</div> )
+);
+jest.mock( 'calypso/components/happiness-support', () => () => (
+	<div data-testid="happiness-support" />
```

Comment:
Nitpicking: For some `data-testid` we use pascal case (see above) and for some - kebab case (like this one). I wonder if it makes sense to be consistent and use the same casing everywhere.

Reply:
Done via https://github.com/Automattic/wp-calypso/pull/66114/commits/99d5536d95a339a0745924d0a7b7d88874ffe20c